test/command/suite/select/function/string/string_slice/nth/options/default_value_without_length.test

```diff
@@ -0,0 +1,10 @@
+plugin_register functions/string
+
+table_create Memos TABLE_HASH_KEY ShortText
+
+load --table Memos
+[
+{"_key": "Groonga"}
+]
+
+select Memos   --output_columns '_key, string_slice(_key, 10, { "default_value" : "default" })'
```

Comment:
How about creating `test/command/suite/select/function/string/string_slice/nth/options/default_value/` directory?

Reply:
I fixed it.